File: 2P/AOO/Revisao/Pessoa.cs
using System;

namespace Revisao
{
    public class Pessoa
    {
        private string nome = "";
        public string Nome
        {
            get { return nome; }
            set { nome = value; }
        }

        private string email = "";
        public string Email
        {
            get { return email; }
            set { email = value; }
        }

    }
}

File: 2P/FPOO/Solucoes_Lista3/Exercicio03/src/main/java/br/edu/vianna/Pessoa.java
package br.edu.vianna;

/**
 * @author William Silva
 */

public class Pessoa {
    protected String nome;
    protected String dataNascimento;

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        if (nome != null && !nome.isBlank()) {
            this.nome = nome;
        }
    }

    public String getDataNascimento() {
        return dataNascimento;
    }

    public void setDataNascimento(String dataNascimento) {
        if (dataNascimento != null && !dataNascimento.isBlank()) {
            this.dataNascimento = dataNascimento;
        }
    }

    @Override
    public String toString() {
        return String.format("Nome: <<>%s> Data Nascimento: <<%s>>", nome, dataNascimento);
    }
}
